Documentation/teaching/lectures/so2.cs.pub.ro.rst

```diff
@@ -0,0 +1,182 @@
+* NeedToKnow: http://ocw.cs.pub.ro/courses/so2/need-to-know
+* mailing list: so2@cursuri.cs.pub.ro
+* Facebook
+* vmchecker
+* catalog Google, calendar Google
+* LXR
+* cs.curs.pub.ro - rol de portal
+* karma awards
+
+Community
+=========
+
+* laboratoarele (din wiki) sunt editabile
+* folosiți contul de pe cs.curs.pub.ro
+* corecții, ajustări, precizări, informații utile
+* cursurile sunt în format Google Docs
+* Pentru corecţii adaugați comentarii
+* pentru detalii folosiți "Speaker Notes"
+* listă de discuții
+* răspundeți la întrebările colegilor voștri
+* propuneți subiecte de discuție care au legătură cu disciplina
+* Facebook
+* sugestii, propuneri, feedback
+* Primiți puncte de karma
+
+Grading
+=======
+
+* 2 puncte activitate la laborator
+* 3 puncte „examen”, notare pe parcurs
+* 10 puncte teme de casă
+* Punctajul > 5 puncte e corelat direct proportional cu nota de la examen (la fel ca la SO)
+* Tema 0 - 0,5 puncte
+* Temele 1, 2, 3 - câte 1,5 puncte fiecare
+* Activități “extra”
+* Ixia challenge - 2 puncte
+* Kernel (filesystem) hackaton - 2 puncte
+* SO2 transport protocol - 1 punct
+* Condiţii de promovare: nota finală 4.5, nota minimă examen 3
+
+Obiectivele cursului
+====================
+
+* Prezentarea structurii interne a unui sistem de operare
+* Target: sisteme de operare de uz general
+* Structura și componentele unui kernel monolitic
+* Procese, FS, Networking
+* Memory management
+* Exemplificare pe Linux
+
+Obiectivele laboratorului/temelor
+=================================
+* Însușirea cunoștințelor necesare implementării de device drivere
+* Înțelegerea în profunzime a cunoștințelor prin rezolvarea de exerciții
+
+Cursuri necesare
+================
+
+* Programare: C
+* SD: tabele de dispersie, arbori echilibrați
+* IOCLA: lucrul cu registre și instrucțiuni de bază (adunări, comparaţii, salturi)
+* CN: TLB/CAM, memorie, procesor, I/O
+* PC, RL: ethernet, IP, sockeți
+* SO: procese, fișiere, thread-uri, memorie virtuală
+
+Despre curs
+===========
+
+* 12 cursuri
+* interactiv
+* participaţi la discuţii
+* întrebaţi atunci când nu aţi înţeles
+* destul de “dens”, se recomandă călduros parcurgerea suportului bibliografic înainte şi după curs
+* 1h:30 prezentare + 30min test si discutii pe marginea testului
+
+Despre curs (2)
+===============
+
+.. hlist::
+   :columns: 2
+   
+   * Introducere
+   * Procese
+   * Scheduling
+   * Apeluri de sistem
+   * Traps
+   * Spaţiul de adresă
+   * Memorie virtuală
+   * Memorie fizică
+   * Kernel debugging
+   * Block I/O
+   * Sisteme de fişiere
+   * SMP
+   * Networking
+   * Virtualizare
+
+
+Despre laborator
+================
+
+* Kernel Modules and Device Drivers
+* 15 min prezentare / 80 de minute lucru
+* se punctează activitatea
+* learn by doing
+
+Despre teme
+===========
+
+* Tema 0
+* Kprobe based tracer
+* Driver pentru portul serial
+* Software RAID
+* Teme “extra”
+* Filesystem driver - hackaton
+* E100 driver - Ixia challenge
```

Comment:
@dbaluta , was that the decision? Let's talk with the assignment team.

Reply:
@razvand  we didn't reach a conclusion on this. My proposal is to use the file system assigment for Ixia challenge.

Reply:
@razvand @dbaluta 
using `fs` for Ixia Challenge is ok considering it was modified by Adrian and lots of solutions for the previous versions are on the internet. We should encourage students to participate in this competition. We could also let students solve it during normal assignment time like scheduled by Razvan. 

I suggest using E100 for hackathon because it is also a complex and useful assignment.

Reply:
@alexandrasandulescu , @dbaluta , I agree to use the `fs` assignment the Ixia challenge and make a hackathon for the `e100` assignment.